kde-apps/kde-l10n/kde-l10n-15.12.3-r2.ebuild

```diff
@@ -100,7 +98,7 @@ EOF
 	if [[ -d ${KHC}/po ]] ; then
 		pushd ${KHC}/po > /dev/null || die
 		for lang in *; do
-			if [[ -d ${lang} ]] && ! has ${lang} ${LINGUAS} ; then
+			if [[ -d ${lang} ]] && ! has ${lang} ${L10N} ; then
```

Comment:
this is looping over linguas directories, so second occurence of ${lang} will need the substitution here as well